apps/web/components/integrations/CalendarListContainer.tsx

```diff
@@ -133,7 +133,10 @@ function CalendarList(props: Props) {
 
 function ConnectedCalendarsList(props: Props) {
   const { t } = useLocale();
-  const query = trpc.useQuery(["viewer.connectedCalendars"], { suspense: true });
+  const query = trpc.viewer.connectedCalendars.useQuery(undefined, {
+    suspense: true,
+    trpc: {},
```

Comment:
What does this `trpc` prop doing in this context?

Reply:
cc @KATT

Reply:
It is for tRPC related options (skipBatch, abortOnUnmount, ssr etc). 

The codemod "accidentally" added this prop even if it's empty but i got rid of most of them (this one must have slipped through).